JobConfig/pionbeam/PiBeamResampler.fcl

```diff
@@ -0,0 +1,79 @@
+#
+# Re-sample pions and associated particles entering the DS, propagate them to the detectors or stopping target, and write out the corresponding flash DetectorSteps and target Pistops.
+# Note the main flash stream has a time cut to avoid energy far outside the digitization window.  A prescaled but otherwise uncut stream is used to
+# record the flash peak
+#
+# original author: Sophie Middleton , 2021, based on code by: Yuri Oksuzkian
+# Updated for MDC2020 (DetectorSteps): D. Brown
+
+#include "Offline/fcl/standardServices.fcl"
+#include "Production/JobConfig/common/prolog.fcl"
+#include "Production/JobConfig/pileup/prolog.fcl"
+
+process_name: PiBeamResampler
+
+source : {
+  module_type : EmptyEvent
+  maxEvents : @nil
+}
+
+services : @local::Services.Sim
+physics: {
+  producers : {
+    @table::Common.producers
+    @table::Pileup.producers
+  }
+  filters : {
+    @table::Common.filters
+    @table::Pileup.filters
+  }
+  analyzers : {
+    @table::Common.analyzers
+  }
+  # setup paths
+  targetStopPath : [ @sequence::Pileup.beamResamplerSequence, @sequence::Common.g4Sequence, TargetStopPrescaleFilter, TargetPionFinder, TargetPiStopFilter, compressPVTargetPiStops]
+  
```

Comment:
You should remove the prescale filter, it makes no sense in a single-stream production.

Reply:
sure

Reply:
Hi @brownd1978 the prescale filter is this: @sequence::Pileup.beamResamplerSequence, right? Removing this has caused it to crash:

```%MSG-s ArtException:  PostEndJob 03-Dec-2021 17:25:14 UTC ModuleEndJob
---- EventProcessorFailure BEGIN
  EventProcessor: an exception occurred during current event processing
  ---- ScheduleExecutionFailure BEGIN
    Path: ProcessingStopped.
    ---- ProductNotFound BEGIN
      Found zero products matching all selection criteria
        C++ type: std::vector<mu2e::StepPointMC>
        Module label: 'beamResampler'
        Product instance name: 'Beam'
        Process name: (empty)The above exception was thrown while processing module Mu2eG4MT/g4run run: 1202 subRun: 0 event: 1
    ---- ProductNotFound END
    Exception going through path targetStopPath
  ---- ScheduleExecutionFailure END
---- EventProcessorFailure END
---- EventProcessorFailure BEGIN
  EventProcessor: an exception occurred during current event processing
  ---- ScheduleExecutionFailure BEGIN
    Path: ProcessingStopped.
    ---- ProductNotFound BEGIN
      Found zero products matching all selection criteria
        C++ type: std::vector<cet::map_vector<mu2e::PhysicalVolumeInfo> >
        Module label: 'beamResampler'
        Product instance name: ''
        Process name: (empty)The above exception was thrown while processing module Mu2eG4MT/g4run run: 1202 subRun: 0
    ---- ProductNotFound END
    Exception going through path targetStopPath
  ---- ScheduleExecutionFailure END
---- EventProcessorFailure END
%MSG```

Reply:
Ah I assume you infact meant: TargetStopPrescaleFilter not the one I removed, that makes more sense, it was cut off in the above quote so I missed it and assumed it was the first option in the sequence